config.rb

```diff
@@ -6,3 +6,23 @@
   activate :minify_css
   activate :minify_javascript
 end
+
+helpers do
+  require "nokogiri"
+  def inline_svg(name, options={})
```

Comment:
Put this together with help from @drapergeek.

Referenced:
* https://robots.thoughtbot.com/organized-workflow-for-svg
* http://cobwwweb.com/render-inline-svg-rails-middleman